migration/src/m20230708_225009_balance_view_v2.rs

```diff
@@ -0,0 +1,48 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        // Replace the sample below with your own migration scripts
+        todo!();
```

Comment:
https://discord.com/channels/873880840487206962/900758302294704188/1127456562865258548

from my searching, it looks like sea_orm doesn't have any official support for Views. This means their tooling won't create the entities for us or help us write the migrations.

Given that it is a read only table and we will just be doing a simple `.filter(User::Id.eq(user_id)` on it, i think we should just add the query select logic to the existing `Balance` model. Given the view isn't a "materialized view", I wonder if we should even bother. We can just write the sql in a  `async fn for_user(id: u64) -> Balance`

Reply:
where would you want to introduce this function?